contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-ios.mdx

```diff
@@ -30,17 +30,41 @@ To handle this, you can use the `didReceiveFeatureFlags` notification to wait fo
 
 ```swift
 // Before SDK initialization
-NotificationCenter.default.addObserver(self, selector: #selector(receiveFeatureFlags), name: PostHogSDK.didReceiveFeatureFlags, object: nil)
-PostHogSDK.shared.setup(config)
+func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool { 
+    // your code
+
+    // Register to didReceiveFeatureFlags notification
+    NotificationCenter.default.addObserver(self, selector: #selector(receiveFeatureFlags), name: PostHogSDK.didReceiveFeatureFlags, object: nil)
+    PostHogSDK.shared.setup(config)
+
+    return true
+}
 
 // The "receiveFeatureFlags" method will be called when the SDK receives the feature flags from the server.
+@objc func receiveFeatureFlags() {
+    print("receiveFeatureFlags called")
+}
+```
+
+Or manually using `reloadFeatureFlags` after the SDK is initialized: 
 
-// And/Or manually the SDK is initialized
-PostHogSDK.shared.reloadFeatureFlags {
-    if PostHogSDK.shared.isFeatureEnabled("flag-key") {
-        // do something
+
+``` swift
+// After SDK initialization
+func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool { 
```

Comment:
I dont think this example should have the `application` and `PostHogSDK.shared.setup(config)` since this is a reload, which can be at any time and can be called multiple times as well